service/history/workflowExecutionContext.go

```diff
@@ -1217,11 +1217,10 @@ func (c *workflowExecutionContextImpl) enforceSizeCheck() (bool, error) {
 	}
 
 	if historySize > historySizeLimitWarn || historyCount > historyCountLimitWarn {
-		executionInfo := c.mutableState.GetExecutionInfo()
```

Comment:
keep this line so below do not have to call `c.mutableState.GetExecutionInfo()` multiple times?

Reply:
2 times. Two times instead of one. Do you think it worth a variable?